package exceptions;

public class Throw1 {
    public static void main(String[] args)
    {
        System.out.println(1/0);
    }
}
